ipa/prover.go

```diff
@@ -206,3 +214,19 @@ func (ip IPAProof) Equal(other IPAProof) bool {
 	}
 	return ip.A_scalar.Equal(&other.A_scalar)
 }
+
+func computeBVector(ic *IPAConfig, evalPoint fr.Element) []fr.Element {
+	if evalPoint.Cmp(&maxEvalPointInsideDomain) > 0 {
+		return ic.PrecomputedWeights.ComputeBarycentricCoefficients(evalPoint)
+	}
```

Comment:
With overwhelming probability, this will be the branch that executions will take.